src/test/java/org/fundacionjala/coding/wilson/VowelCounterTest.java

```diff
@@ -0,0 +1,38 @@
+package org.fundacionjala.coding.wilson;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Some text here.
+ */
+
+public class VowelCounterTest {
+
+    /**
+     *
+     */
+    @Test
+    public void testNull() {
+        final int expected = 0;
+        assertEquals("Nope!", expected, VowelCounter.vowelCount(""));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Nope!" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)